opencti-platform/opencti-front/src/private/components/common/stix_core_objects/StixCoreObjectFileExport.jsx

```diff
@@ -0,0 +1,265 @@
+
+const stixCoreObjectFileExportQuery = graphql`
+  query StixCoreObjectFileExportQuery {
+    connectorsForExport {
+      id
+      name
+      active
+      connector_scope
+      updated_at
+    }
+  }
+`;
+
+// TODO see if possible to extract common logic between this component
+// and StixCoreObjectsExportCreation to limit duplicated code.
+
+const exportValidation = (t) => Yup.object().shape({
+  format: Yup.string().required(t('This field is required')),
+});
+
+export const scopesConn = (exportConnectors) => {
+  const scopes = R.uniq(
+    R.flatten(R.map((c) => c.connector_scope, exportConnectors)),
+  );
+  const connectors = R.map((s) => {
+    const filteredConnectors = R.filter(
+      (e) => R.includes(s, e.connector_scope),
+      exportConnectors,
+    );
+    return R.map(
+      (x) => ({ data: { name: x.name, active: x.active } }),
+      filteredConnectors,
+    );
+  }, scopes);
+  const zipped = R.zip(scopes, connectors);
+  return R.fromPairs(zipped);
+};
+
+const StixCoreObjectFileExportComponent = ({
+  queryRef,
+  id,
+}) => {
+  const navigate = useNavigate();
+  const { t } = useFormatter();
+
+  const data = usePreloadedQuery(
+    stixCoreObjectFileExportQuery,
+    queryRef,
+  );
+
+  const [open, setOpen] = useState(false);
+
+  const onSubmitExport = (values, { setSubmitting, resetForm }) => {
+    const maxMarkingDefinition = values.maxMarkingDefinition === 'none'
+      ? null
+      : values.maxMarkingDefinition;
+    commitMutation({
+      mutation: fileManagerExportMutation,
+      variables: {
+        id,
+        format: values.format,
+        exportType: values.type,
+        maxMarkingDefinition,
+      },
+
+      onCompleted: (exportData) => {
+        const fileId = exportData.stixCoreObjectEdit.exportAsk[0].id;
+        setSubmitting(false);
+        resetForm();
+        MESSAGING$.notifySuccess('Export successfully started');
+        navigate({
+          pathname: `/dashboard/analyses/reports/${id}/content`,
+          search: `?${createSearchParams({ currentFileId: fileId })}`,
+        });
+      },
+    });
+  };
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const connectorsExport = R.propOr([], 'connectorsForExport', data);
+  const exportScopes = R.uniq(
+    R.flatten(R.map((c) => c.connector_scope, connectorsExport)),
+  );
+
+  const formatValue = exportScopes[0];
+
+  const exportConnsPerFormat = scopesConn(connectorsExport);
+  const isExportActive = (format) => R.filter((x) => x.data.active, exportConnsPerFormat[format]).length > 0;
+  const isExportPossible = R.filter((x) => isExportActive(x), exportScopes).length > 0;
+
+  return (
+    <div>
+      <Tooltip
+        title={
+          isExportPossible
+            ? t('Generate an export')
+            : t('No export connector available to generate an export')
+        }
+        aria-label="generate-export"
+      >
+        <ToggleButton
+          onClick={() => handleClickOpen()}
```

Comment:
onClick={handleClickOpen}

Reply:
ESLint warning was rised by doing this because onClick callback have params but hadnClickOpen does not